### DISCUSSION TITLE
Public website private repo

### DISCUSSION THREAD

**Discussion by dylwil3** [community/community#35902] [Other Feature Feedback, Questions, & Ideas] [answered]
I would like to have a private repo but a public website. I have GitHub premium. I do not have enterprise.

(I tried setting my repo to private. My website disappeared. It reappeared after resetting it to public and making a commit.)

Note: there is a similar question, but the "answer" magically answers a different question, and states that if you have a free account and make the repo private then *both* the repo and the website become invisible. That is not the situation of the current question.

**Comment by MitkoVtori** [comment, marked as answer]
If you have premium it should work. 
try finding help [here](https://docs.github.com/en/pages/getting-started-with-github-pages/about-github-pages)

If this help you mark this as answer so nobody will waste his time.

**Reply by dylwil3** [reply]
Okay, I tried it again and it worked- I think the upgrade to GitHub premium had not processed yet the first time. Thanks!
